### DISCUSSION TITLE
Video explanations for questions

### DISCUSSION THREAD

**Discussion by gfeeng** [greatfrontend/greatfrontend#29] [Ideas/Feature Requests]
> For each coding and quiz, it will be great to have a video explanation.  Regardless, I am enjoying the service.

**Comment by sicktastic** [comment]
Can I create one for free questions?

**Reply by gfeeng** [reply]
Yes! Go ahead and do so. If GreatFrontEnd's descriptions, solutions, tests are shown, source/credit should be given where it's appropriate.
